execution_engine/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ All notable changes to this project will be documented in this file.  The format
 * Support building and testing using stable Rust.
 * Increase price of `create_purse` to 2.5CSPR.
 * Improve doc comments to clarify behavior of the bidding functionality.
+* Documented `core` module and children.
```

Comment:
Is it all `core` modules or only the EE one?

Reply:
Only core from EE, but this changelog is specific to EE, so should be fine as is I think